src/bundle/SystemInfo/Collector/JsonComposerLockSystemInfoCollector.php

```diff
@@ -171,4 +174,25 @@ private static function setNormalizedVersion(ComposerPackage $package): void
 
         $package->version = $version;
     }
+
+    private function getStability(array $lockData): ?string
+    {
+        foreach ($lockData['packages'] as $package) {
+            if ($package['name'] === self::IBEXA_OSS_PACKAGE) {
+                return $this->getIbexaOssStability($package['version']);
+            }
+        }
+
+        return $this->getMinimumStability($lockData);
+    }
+
+    private function getIbexaOssStability(string $version): string
+    {
+        return $version === self::DEV_VERSION ? self::STABILITIES[20] : self::STABILITIES[0];
```

Comment:
This is neither stable nor complete. As you can see `self::STABILITIES` contains multiple stability constraints. For RC packages we need to display "RC", for Beta - "Beta" and so on.
Moreover while right `$version === 'dev-master'` is correct for the latest release, for earlier ones it will become e.g. `3.2.x-dev` or `3.4.x-dev`. To retrieve that information we should rather use PHP API from `composer/composer` package. I wonder if it contains some more abstract way to get a stability of a package - TBD.